lib/chef/cookbook/cookbook_version_loader.rb

```diff
@@ -88,12 +75,17 @@ def load
         remove_ignored_files
 
         if empty?
-          Chef::Log.warn "Found a directory #{cookbook_name} in the cookbook path, but it contains no cookbook files. skipping."
+          raise Exceptions::CookbookNotFoundInRepo, "The directory #{cookbook_path} does not contain a cookbook"
         end
+
         cookbook_settings
       end
 
-      alias :load_cookbooks :load
+      def load
+        Chef::Log.warn "load method is deprecated. Use load! instead"
```

Comment:
that should really raise here as well.  also `load_cookbooks` should still be aliased to `load` for this test.

Reply:
@lamont-granquist I have done the changes so that we can see the failing test.